litmus-portal/backend/graphql-server/build/Dockerfile

```diff
@@ -1,25 +1,20 @@
-FROM golang:1.14.4-alpine
+# BUILD STAGE
```

Comment:
As it is a small docker file, we don't need the multi stage build

Reply:
Can you put the difference of the image size

Reply:
Yes, it is decreasing the size. We can have the multi-stage build. I ran with  CGO_ENABLED=0, it is running fine